api/graphql/mutations.js

```diff
@@ -0,0 +1,14 @@
+import {
+  convertGraphqlCreateData,
+  createPost as underlyingCreatePost,
+  validateGraphqlCreateData
+} from '../models/post/util'
+
+export const updateMe = (userId, changes) =>
```

Comment:
i think we want to get in the habit of doing `export function` so that we create named functions, which is better for debugging